barretenberg/cpp/src/barretenberg/benchmark/ultra_bench/ultra_honk_rounds.bench.cpp

```diff
@@ -63,7 +64,10 @@ BB_PROFILE static void test_round(State& state, size_t index) noexcept
     GoblinUltraProver prover = bb::mock_proofs::get_prover(
         composer, &bb::mock_proofs::generate_basic_arithmetic_circuit<GoblinUltraCircuitBuilder>, log2_num_gates);
     for (auto _ : state) {
+        state.PauseTiming();
         test_round_inner(state, prover, index);
+        state.ResumeTiming();
+        // NOTE: google bench is very finnicky, must end in ResumeTiming() for correctness
```

Comment:
not sure why this was removed in the protogalaxy rounds bench. This is required for google bench to function.

Reply:
hmm, this is not strictly related to this PR and we should be a bit more conservative withchanges to benchmarks so maybe address in a follow-up?

Reply:
sure, I can just make a separate PR for this